include/zephyr/linker/intlist.ld

```diff
@@ -18,29 +18,10 @@
  * defined, the total number of IRQ lines in the system, followed by
  * an appropriate number of instances of struct _isr_list. See
  * include/sw_isr_table.h
- *
- * You will need to declare a bogus memory region for IDT_LIST. It doesn't
- * matter where this region goes as it is stripped from the final ELF image.
- * The address doesn't even have to be valid on the target. However, it
- * shouldn't overlap any other regions. On most arches the following should be
- * fine:
- *
- * MEMORY {
- *	.. other regions ..
- *	IDT_LIST        : ORIGIN = 0xfffff7ff, LENGTH = 2K
- * }
  */
 
-#ifndef LINKER_ZEPHYR_FINAL
-SECTION_PROLOGUE(.intList,,)
-{
-	KEEP(*(.irq_info*))
-	KEEP(*(.intList*))
-} GROUP_ROM_LINK_IN(IDT_LIST, IDT_LIST)
-#else
-/DISCARD/ :
+SECTION_PROLOGUE(.intList, 0 (INFO),)
```

Comment:
Unfortunately, this still needs to be discarded during final linking. The `rimage` stitching and signing tool for audio considers this as a real section where it complains about overlapping sections (as the address of this section is meaningless so the tool somehow assigns the base address => overlapping with other sections). So for now, please keep the DISCARD in place.

Reply:
Updated.